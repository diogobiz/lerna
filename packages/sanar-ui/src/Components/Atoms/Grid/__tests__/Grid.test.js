import React from 'react'
import TestRenderer from 'react-test-renderer'

import { ESRow, ESCol } from '../'

it('renders correctly', () => {
    const component = (
        <>
            <ESRow>
                <ESCol span={12}>col-12</ESCol>
                <ESCol span={12}>col-12</ESCol>
            </ESRow>
            <ESRow>
                <ESCol span={8}>col-8</ESCol>
                <ESCol span={8}>col-8</ESCol>
                <ESCol span={8}>col-8</ESCol>
            </ESRow>
            <ESRow>
                <ESCol span={6}>col-6</ESCol>
                <ESCol span={6}>col-6</ESCol>
                <ESCol span={6}>col-6</ESCol>
                <ESCol span={6}>col-6</ESCol>
            </ESRow>
        </>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
