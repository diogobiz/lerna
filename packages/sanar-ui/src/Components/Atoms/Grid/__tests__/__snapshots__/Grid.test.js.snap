// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="ant-row es-row"
    style={Object {}}
  >
    <div
      className="ant-col ant-col-12 es-col"
      style={Object {}}
    >
      col-12
    </div>
    <div
      className="ant-col ant-col-12 es-col"
      style={Object {}}
    >
      col-12
    </div>
  </div>,
  <div
    className="ant-row es-row"
    style={Object {}}
  >
    <div
      className="ant-col ant-col-8 es-col"
      style={Object {}}
    >
      col-8
    </div>
    <div
      className="ant-col ant-col-8 es-col"
      style={Object {}}
    >
      col-8
    </div>
    <div
      className="ant-col ant-col-8 es-col"
      style={Object {}}
    >
      col-8
    </div>
  </div>,
  <div
    className="ant-row es-row"
    style={Object {}}
  >
    <div
      className="ant-col ant-col-6 es-col"
      style={Object {}}
    >
      col-6
    </div>
    <div
      className="ant-col ant-col-6 es-col"
      style={Object {}}
    >
      col-6
    </div>
    <div
      className="ant-col ant-col-6 es-col"
      style={Object {}}
    >
      col-6
    </div>
    <div
      className="ant-col ant-col-6 es-col"
      style={Object {}}
    >
      col-6
    </div>
  </div>,
]
`;
